src/components/SkillPage/SkillListing.js

```diff
@@ -219,6 +236,20 @@ class SkillListing extends Component {
         })
     }
 
+    saveCountryWiseSkillUsage = (country_wise_skill_usage = []) => {
+        // Add sample data to test
+        const data = [];
+
+        for (var i = 0; i < country_wise_skill_usage.length; i++) {
```

Comment:
I think you can use either some better loop like `for .. of` or `for .. in` or I think `.map()` should work too. :thinking:

Reply:
The complexity remains the same.

Reply:
Yes, though it's advisable to follow the ES6 Syntax.

Reply:
Ok